server/jetstream_cluster_4_test.go

```diff
@@ -1129,6 +1132,8 @@ func TestJetStreamClusterStreamOrphanMsgsAndReplicasDrifting(t *testing.T) {
 							for range time.NewTicker(2 * time.Second).C {
 								select {
 								case <-hctx.Done():
+									t.Fatalf("Timeout checking for health of %s", s.Name())
```

Comment:
think this line above needs fixing: `s = c.restartServer(s)`

Reply:
Thank you 👍